### PR TITLE
Add Single Click on Win OS Server System Tray Icon

### LINKED ISSUE #1722: Add Click action to Server System Tray Icon when using Windows OS
**Has this feature been discussed and generally agreed?**
Not discussed yet, but happy to do here as it is not really complex.

**Describe the solution you'd like**
Working on Windows with application in the systemtray, it is very common these days that a single (left) click on the tray icon opens a dialog or context menu. Current when running Jamulus with the server option, nothing happens with single click.
Double Click is implemented, but not intuitive for most people these days. Right Click is implemented and shows a context menu.
On Windows I would like to show the main server dialog when using the single (left) click on the tray icon.

**To consider**
On Linux, the single (left) click is showing the context menu.
Not sure about macOS.
Hence why the feature request is for Windows OS only.

### PR DESCRIPTION
**Pull request by henkdegroot**
Fixes #1722

### CODE REVIEW

File: src/serverdlg.cpp [hunk 1]
```diff
@@ -598,6 +598,14 @@ void CServerDlg::OnSysTrayActivated ( QSystemTrayIcon::ActivationReason ActReaso
     {
         ShowWindowInForeground();
     }
+
```
Comment (pljones) [inline src/serverdlg.cpp:601]: I'd prefer
```
#ifdef _WIN32
if ( ActReason == QSystemTrayIcon::Trigger || ActReason == QSystemTrayIcon::DoubleClick )
#else
if ( ActReason == QSystemTrayIcon::DoubleClick )
#endif
```
over the duplication - that's partly why the code format keeps the `if` on its own line, I think.

Comment (henkdegroot) [reply src/serverdlg.cpp:601]: Thank you for the feedback, indeed makes sense to do so.  Will update, test and update the PR.

**Review by pljones — APPROVED** [review]

**Review by ann0see — APPROVED** [review]
Works as expected

**Comment by hoffie** [comment]
CHANGELOG: GUI: Single clicking Jamulus' server system tray icon will now show the UI on Windows as well.

**Pull request merged**